server/models/User.coffee

```diff
@@ -463,7 +463,7 @@ UserSchema.statics.editableProperties = [
   'testGroupNumber', 'music', 'hourOfCode', 'hourOfCodeComplete', 'preferredLanguage',
   'wizard', 'aceConfig', 'autocastDelay', 'lastLevel', 'jobProfile', 'savedEmployerFilterAlerts',
   'heroConfig', 'iosIdentifierForVendor', 'siteref', 'referrer', 'schoolName', 'role', 'birthday',
-  'enrollmentRequestSent'
+  'enrollmentRequestSent', 'signupQueryParams'
```

Comment:
Well, actually adding an index should go hand-in-hand with setting up an endpoint to use it. There's not really a good place to put a comment ahead of this, I think.